.github/workflows/ci-go.yaml

```diff
@@ -421,9 +421,22 @@ jobs:
     env:
       ATLAS_ACTION_LOCAL: 1
     steps:
+      # views are logged in only feature in atlas
```

Comment:
Are you cehcking the result somewhere to test the views are included? If the cli is not logged in there won't be an error, it will just ignore the view.

Reply:
it the cli failed to login the action will fail, I dont have way to programatically check that view was included